erpnext/domains/hospitality.py

```diff
@@ -1,35 +0,0 @@
-data = {
-	'desktop_icons': [
-		'Restaurant',
-		'Hotels',
-		'Accounts',
-		'Buying',
-		'Stock',
-		'HR',
-		'Project',
-		'ToDo'
-	],
-	'restricted_roles': [
-		'Restaurant Manager',
-		'Hotel Manager',
-		'Hotel Reservation User'
-	],
-	'custom_fields': {
```

Comment:
These custom fields should also be deleted in the patch

Reply:
raised PR: https://github.com/frappe/erpnext/pull/29382

Reply:
Thanks, @ankush 

Added this point to the checklist as well. https://frappe.io/handbook/splitting-modules @ruchamahabal